### ISSUE TITLE
[A11Y] - All Reader View | Search page results should be changed from h3 to h2

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12663]
## Issue Description
On the OPS  ... However, on the Search page, it is recommended that the results be Heading 2s and not Heading 3s. This should be remediated. 

## How to reproduce / Current behavior
Navigate to the search page and enter an input that returns results.  

## What application are you using?
 OJS, OMP or OPS version 3.5.0-4

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, using screenreader

## Action
change h3 to h2 for search results.

## Additional information
WCAG rules : [1.3.1 info and relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) A and [2.4.6 Headings and labels](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels.html) AA.  Recommended for best practice to have search results for h2.

<img width="1137" height="109" alt="Image" src="https://github.com/user-attachments/assets/3d4f78dd-8929-47d4-a7d1-d5bf0277c384" />

<img width="638" height="278" alt="Image" src="https://github.com/user-attachments/assets/afd11803-10df-4e2a-af6f-062508115280" />
